src/app/drive/components/MoveItemsDialog/MoveItemsDialog.tsx

```diff
@@ -129,8 +132,15 @@ const MoveItemsDialog = (props: MoveItemsDialogProps): JSX.Element => {
     dispatch(setItemsToMove([]));
   };
 
-  const setDriveBreadcrumb = async (itemsToMove) => {
-    const breadcrumbsList: FolderAncestor[] = await newStorageService.getFolderAncestors(itemsToMove[0].uuid);
+  const setDriveBreadcrumb = async (itemsToMove: DriveItemData[]) => {
+    const item = itemsToMove[0];
+    const itemUuid = item.uuid;
+    const itemFolderUuid = item.isFolder ? itemUuid : item.folderUuid;
+    const itemType = item.isFolder ? 'folder' : 'file';
+    const token = localStorage.getItem(`${itemType}AccessToken`) || undefined;
```

Comment:
there is local storage service, add functions to get the `FileAccessToken` or `FolderAccessToken` directly there, so it will be easier to maintain.
Change the use of localStorage in all other places that is required these tokens